rafiki/custom_models_base/darknet/cfg/food/food.data

```diff
@@ -0,0 +1,4 @@
+classes= 1
```

Comment:
remove the foodlg.
it is confidential. so we can upload the model into the deployed version, but we cannot upload the model and data into the repo which is public.
in addition, foodlg models are just example models. why put them under custom_models_base?

Reply:
ok,